### PR TITLE
Fix potential nil on transfer

### PR DESCRIPTION
**Pull request by Garanas**
Both `GetReclaimablesInRect` and `GetUnitsInRect` return nil if nothing is found, instead of an empty table. This can happen, as it did during [this replay](https://replay.faforever.com/15419462) at around 35:00 to LilMiss.

image

### CODE REVIEW

File: lua/SimUtils.lua [hunk 1]
```diff
@@ -379,16 +379,22 @@ function TransferUnfinishedUnitsAfterDeath(units, armies)
                 failedToTransfer[ID].Orientation = unit:GetOrientation()
 
 
-                for _, reclaim in GetReclaimablesInRect(unit:GetSkirtRect()) do --wrecks can prevent drone from starting construction
-                    if reclaim.IsWreckage then
-                        reclaim:SetCollisionShape('None') --so we set collision shape 'None'
-                        table.insert(modifiedWrecks, reclaim) --and save wrecks to revert our changes later
+                local wrecks = GetReclaimablesInRect(unit:GetSkirtRect())
```
Comment (speed2CZ) [inline lua/SimUtils.lua:382]: for _, reclaim in GetReclaimablesInRect(unit:GetSkirtRect()) or {} do

Comment (Garanas) [reply lua/SimUtils.lua:382]: I find the current solution to be more readable.

File: units/UES0103/UES0103_script.lua [hunk 2]
```diff
@@ -5,7 +5,7 @@
 --#**
 --#**  Summary  :  UEF Frigate Script
 --#**
---#**  Copyright � 2005 Gas Powered Games, Inc.  All rights reserved.
+--#**  Copyright � 2005 Gas Powered Games, Inc.  All rights reserved.
```
Comment (Sheikah45) [inline units/UES0103/UES0103_script.lua:8]: Unintended encoding change

Comment (Garanas) [reply units/UES0103/UES0103_script.lua:8]: Yep, my laptop has issues with that 😞

**Pull request merged**